std/algorithm/sorting.d

```diff
@@ -774,8 +774,54 @@ private template validPredicates(E, less...)
             && validPredicates!(E, less[1 .. $]);
 }
 
+struct MultiPred( preds... )
```

Comment:
Do we actually want this to be accessible? What would be other use cases for MultiPred?
I'd favor `MultiSortPred` btw, to emphasize how the multiple predicates are used, `MultiPred` should like an `&&` combinator.